hasher-matcher-actioner/hmalib/lambdas/api/collabs.py

```diff
@@ -56,17 +62,19 @@ def get_available_schemas():
 
         A future version could pull from a static location or scan all packages
         for subclasses.
-
-        PS: Is schemata the correct plural?
         """
-        configs = tx_collab_config.get_all_collab_configs()
+        apis = tx_apis.ToggleableSignalExchangeAPIConfig.get_all()
+        config_classes = [
+            api.to_concrete_class().get_config_cls()
+            for api in apis
+            # Do not want to handle local files without supporting s3 uploads
+            # for those files..
```

Comment:
We'd probably just implement S3 exchange as its own style in hmalib (or as an extension).